test/interactions/CTKNCowdsale-setDollarRate_test.js

```diff
@@ -22,23 +22,27 @@ contract('CTKNCrowdsale setDollarRate', accounts => {
   })
 
   context('given bad data', () => {
-    it('throws if rate is 0', () => assertThrows(crowdsale.setDollarRate(0)))
+    it('throws if rate is 0', () =>
+      assertThrows(crowdsale.setUSDConversionRate(0)))
   })
 
   context('given good data', () => {
-    const dollarRate = toWei(0.75)
+    // let's change it so 1 ETH is US$1000 (nice default)
+    // ie 1 dollar buys 0.001 ETH
+    // 1 cent buys 0.00001 ETH
+    const usdConversionRate = toWei(0.00001)
```

Comment:
const centsToWei = toWei(0.00001) ?

Reply:
`cents` could be one of a hundred currencies